workshop/4-Messaging/4.1-Pinpoint.ipynb

```diff
@@ -691,7 +691,7 @@
     "\n",
     "![Create Retail Demo Store account](images/retaildemostore-create-account.png)\n",
     "\n",
-    "3. A few seconds after creating your account you should receive an SMS with a six digit confirmation code. Enter this code on the confirmation page.\n",
+    "3. A few seconds after creating your account you should receive an email with a six digit confirmation code. Enter this code on the confirmation page.\n",
```

Comment:
This notebook needs to be updated to include all the functionality in the pre-create lambda function. In other words, creating the segment, campaign, and message template for the SMS functionality.

Reply:
Yes, I will update the workshop.